curiouslearning/FeedTheMonsterJS
# feat: Adjusted Stone Positions to avoid overlapping on Rive Monster

# Changes
- Stone handler cleaned up and hooked on the game state service.
- Game state how has a parameter for Rive width and sizes (Not yet fully set but ready to be updated in the future for updated Rive animation), and now returns a DAO for the stone coordinates.
- Data acccess object now handles the stone coordinates config setup and randomized that was originally handled in stone-handler.ts but now the logic is moved here and accessed via get methods from game state service.
- Code clean up with some space formatting and comments.

# How to test
- Start the game and select any level on level selection.
- During the game play, no stone letters should overlap or too close with the current rive monster implemented.

Ref: [FM-266](https://curiouslearning.atlassian.net/browse/FM-266)


[FM-266]: https://curiouslearning.atlassian.net/browse/FM-266?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
